.pre-commit-config.yaml

```diff
@@ -23,16 +23,13 @@ repos:
     rev: 5.8.0
     hooks:
       - id: isort
+        args: [ "--profile", "black" ]
   - repo: https://github.com/humitos/mirrors-autoflake.git
     rev: v1.1
     hooks:
       - id: autoflake
         args: ["--in-place", "--remove-all-unused-imports"]
-  - repo: https://github.com/psf/black
```

Comment:
did you find any conflicts with these 2?

Reply:
yes. isort has a black feature that I enabled